features/step_definitions/tag_set_steps.rb

```diff
@@ -9,7 +9,7 @@
 # If you want ratings, warnings, or categories, first load basic or default tags for those types
 When /^I add (.*) to the tag ?set$/ do |tags|
   tags.scan(/the (\w+) tags "([^\"]*)"/).each do |type, scanned_tags|
-    if type == "category" || type == "rating" || type == "warning"
+    if type == "category" || type == "rating" || type == "archive_warning"
```

Comment:
We'll still be referring to the tags as warning tags, so I think it would be more intuitive to make any adjustments we need to allow us to continue writing the step as `When I add the archive_warning tags "No Archive Warnings Apply" to the tag set "Archive Warnings"`.

Reply:
Hi @sarken, I believe this is the case as seen on tag_set.feature line 104 (https://github.com/littlelines/otwarchive/blob/AO3-5613_ruby_rename_warning/features/tag_sets/tag_set.feature#L104) We are able to write features as you described. Is that what you mean?

Reply:
Sorry, I copy/pasted the line and forgot to edit it. That's embarrassing. /o\

I meant we'd like to keep writing the step as `When I add the warning tags "No Archive Warnings Apply" to the tag set "Archive Warnings"`, without the `archive_`. The same is also true for these two steps:
- `And I set up the tag set "Warnings" with the warning tags "Choose Not To Use Archive Warnings"`
- `When I remove the warning tags "Choose Not To Use Archive Warnings" from the tag set "Warnings"`